objects/rct2tt/ride/rct2.tt.bmvoctps.json

```
@@ -150,6 +151,7 @@
             "sv-SE": "En roterande observationskabin som färdas upp i ett högt torn.",
             "ko-KR": "높은 탑을 따라 올라가면서 회전하는 전망대 객실",
             "zh-TW": "主題觀察艙旋轉移動到高塔上面",
+            "ja-JP": "展望キャビンが回転しながら高い塔を登りますが、宇宙からの塊の形があります",
```

The last part is a bit off. Let's go with the following, in line with the other translations:
```suggestion
            "ja-JP": "展望キャビンが回転しながら高い塔を登ります",
```